build.sbt

```diff
@@ -158,6 +158,11 @@ lazy val `kamon-instrumentation-common` = (project in file("instrumentation/kamo
   .disablePlugins(AssemblyPlugin)
   .enablePlugins(JavaAgent)
   .settings(instrumentationSettings)
+  .settings(
+        crossScalaVersions += `scala_3_version`,
+  ).settings(
+    scalacOptions ++= { if(scalaBinaryVersion.value.startsWith("3.")) Seq("-Xtarget:8") else Seq.empty }
```

Comment:
Can you please move this setting to this other file: https://github.com/kamon-io/Kamon/blob/master/project/Build.scala#L167